### DISCUSSION TITLE
Data type `array` support

### DISCUSSION THREAD

**Discussion by lxw-mesoor** [milvus-io/milvus#4245] [Q&A and General discussion] [answered]
Will it support `array` data type? 
[Elasticsearch arrays](https://www.elastic.co/guide/en/elasticsearch/reference/current/array.html)

**Comment by shiyu22** [comment]
Actually, Milvus is going to do it, are you talking about [inserting entity like this](https://pymilvus.readthedocs.io/en/latest/tutorial.html#insert-entities)?
If not, please list examples.

**Reply by lxw-mesoor** [reply]
For example, there are entities like 
`{"labels": [1, 2, 3], "tags": 1, "embedding": [0.1, 0.2, 0.3]}`
`{"labels": 1, "tags": [1, 2, 3], "embedding": [0.1, 0.2, 0.3]}`
It seems Milvus only supports entities like 
`{"labels": 1, "tags": 1, "embedding": [0.1, 0.2, 0.3]}`
Am I right?

**Comment by yhmo** [comment, marked as answer]
I create an issue #4264 for this.
